### PR TITLE
Adding a new file

### PR DESCRIPTION
**Pull request by mnlevy1981**
Just trying to test the new github PR template location (and also see if the checklist action is being run)

### All Submissions:

* [x] Have you followed the guidelines in our Contributer's Guide](https://github.com/NCAR/CUPiD/wiki/Contributor's-Guide) (including the `pre-commit` check)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Comment by mnlevy1981** [comment]
The checklist task is failing because

```
Run kentaro-m/task-completed-checker-action@v0.1.2
  with:
    repo-token: ***
Error: Resource not accessible by integration
```

File: .github/workflows/tasklist_completed.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ on:
 jobs:
   task-check:
     runs-on: ubuntu-latest
+    permissions:
+      checks: write
```
Comment (mnlevy1981) [inline .github/workflows/tasklist_completed.yml:10]: I think updating a workflow yml file in a PR will run the version of the test from `main` -- I'm still seeing

```
GITHUB_TOKEN Permissions
  Checks: read
  Metadata: read
```

When checks should be "write". I'm tempted to make this change directly on `main` and re-run the test

Comment (TeaganKing) [reply .github/workflows/tasklist_completed.yml:10]: Seems reasonable to me!

**Comment by mnlevy1981** [comment]
Closing this, will try to start a new PR from updated `main` with current permissions

**Pull request closed without merging**